Gemfile

```diff
@@ -45,13 +45,13 @@ gem 'sidekiq'
 gem 'bluecloth'
 
 # Pagination
-gem 'will_paginate', '~> 3.0'
+gem 'will_paginate'
 
 # user signup/login/etc
-gem 'devise', '>= 4.0.0'
```

Comment:
I think >= pins should be retained; so that other gems can't downgrade us if they aren't compatible (we get a bundler error instead)